governance/Coding-Standards-and-Style-Guide.md

```diff
@@ -132,7 +132,7 @@ For reference purposes C++17 feature support information can be found here:
 4\. Formatting
 ==============
 
-Rather than specify detailed formatting guidelines in this document we are using a code formatting tool (Uncrustify) that converts code to the desired formatting. See this document for details of how to use it: [Using Uncrustify to beautify Lumberyard code](/display/lmbr/Using+Uncrustify+to+beautify+Lumberyard+code)
+Rather than specify detailed formatting guidelines in this document we are using a code formatting tool (Clang-format) that converts code to the desired formatting.  See the .clang-format file at the root of the reopsitory.
```

Comment:
Typo in "reopsitory"

```suggestion
Rather than specify detailed formatting guidelines in this document we are using a code formatting tool (Clang-format) that converts code to the desired formatting.  See the .clang-format file at the root of the repository.
```

Reply:
will fix!